README.md

```diff
@@ -292,6 +340,38 @@ app.after(async function () {
 
 Returns the instance on which `after` is called, to support a chainable API.
 
+-------------------------------------------------------
+<a name="await-after"></a>
+
+### await app.after() | app.after() => Promise
+
+Calling after with no function argument loads any plugins previously registered via `use` and returns a promise which resolves when all plugins registered so far have loaded.  
```

Comment:
Should we specify that `await after()` will never reject?

Reply:
I actually _missed_ that this would never reject. I'm not really convinced of this behavior.
Should it reject instead?